pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -0,0 +1,40 @@
+node('jenkins-master') {
+  dir('jenkins-dm-jobs') {
+    checkout([
+      $class: 'GitSCM',
+      branches: scm.getBranches(),
+      userRemoteConfigs: scm.getUserRemoteConfigs(),
+      changelog: false,
+      poll: false
+    ])
+    notify = load 'pipelines/lib/notify.groovy'
+    util = load 'pipelines/lib/util.groovy'
+  }
+}
+
+notify.wrap {
+  node('docker') {
+    timeout(time: 4, unit: 'HOURS') {
+      dir('imgserv') {
+        git([
+          url: util.githubSlugToUrl('lsst/dax_imgserv'),
+          branch: 'master'
+        ])
+      }
+
+      build('pro_image.sh')
```

Comment:
The git repo will be checked out as `./imgserv` but this is going to try to run `pro_image.sh` one directory higher (which would presumably fail).

Reply:
Fixed.